### PR TITLE
add Avatar to Member on Google oAuth

### PR DESCRIPTION
**Pull request by WengerK**
### 💬 Describe the pull request
- add Avatar to Member on Google oAuth
- Ensure Avatar is fetched via a custom Controller to prevent sending referee and data to Google on image download

@gido can you review my fetcher process ? Does it works as it should ? Is it sufficient ?

### CODE REVIEW

File: src/Controller/AvatarController.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+     * @Route("/avatar/{id}", name="app_avatar_fetch")
+     */
+    public function fetch(User $user): ?RedirectResponse
+    {
+        $avatar = $user->getAvatar();
+        if (!$avatar) {
+            return null;
+        }
+
+        return $this->redirect($avatar);
```
Comment (gido) [inline src/Controller/AvatarController.php:25]: OK, if we want to only prevent from referrer leakage we can set a [Referrer-Policy](https://web.dev/referrer-best-practices/) by adding the following header on each request response:

```
Referrer-Policy: same-origin
```

So we don't need anymore this controller and you can add the meta tag in the head of the page template or a HTTP header

File: templates/base.html.twig [hunk 2]
```diff
@@ -50,8 +50,13 @@
                         <div class="navbar-nav flex-row order-md-last">
                             <div class="nav-item">
                                 <div class="nav-link d-flex lh-1 text-reset p-0">
-                                <span class="avatar avatar-sm" data-bs-toggle="tooltip" data-bs-placement="bottom"
-                                      title="{{ app.user }}">{{ app.user.getInitials }}</span>
+                                    <span class="avatar avatar-sm" data-bs-toggle="tooltip" data-bs-placement="bottom"
+                                          title="{{ app.user }}"
+                                          style="background-image: url({{ path('app_avatar_fetch', {id: app.user.id}) }})">
```
Comment (gido) [inline templates/base.html.twig:55]: Add a if avatar

File: templates/projects/list_project.html.twig [hunk 3]
```diff
@@ -155,7 +155,12 @@
                                                 {% for member in project.team %}
                                                     <span class="avatar avatar-sm avatar-rounded" data-bs-toggle="tooltip"
                                                           data-bs-placement="top"
-                                                          title="{{ member.user }}">{{ member.user.initials }}</span>
+                                                          title="{{ member.user }}"
+                                                          style="background-image: url({{ path('app_avatar_fetch', {id: member.user.id}) }})">
```
Comment (gido) [inline templates/projects/list_project.html.twig:159]: Same comment

**Review by gido — CHANGES_REQUESTED** [review]

**Comment by WengerK** [comment]
Thanks I made all the changes !

**Review by gido — APPROVED** [review]

**Comment by gido** [comment]
Thanks @WengerK !

**Pull request merged**